_drafts/why-code.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Why learn to code?"
+author: Lauren Wolfe
+categories: # delete the categories that do not apply and keep at least one
+  - Community
+tags: # add 1-4 lowercase tags that are relevant to your post, ex: r, python, genomics, workflows
+  - r
+  - python
+  - training
+  - coding
+---
+Chances are, that if you're a biologist in the 21st century, you have been told at some point that you should learn to code. In fact, when I graduated from Western Washington University in 2014 it was said to me! So, shortly after graduation I took my first coding class. It was in C++ and I made a `hello world` script and some calculators. I walked away excited that I had learned something new but with no idea how this would one day apply to the biology work I hoped to do. Software and coding in biology tends to be associated with cutting-edge projects that require in-depth computational and statistical knowledge and access to high-powered compute environments. When all we hear about are the cutting-edge, paradigm-shifting ways that code is used to make biological insights it leaves beginners wondering when, if ever, the skillset will become useful to them. However, even a more basic knowledge of coding can improve research and make mundane lab tasks quicker and more effective.
```

Comment:
One of the most common questions we receive at The Coop is whether someone in our community should learn to code. Changes are good that you've heard someone talk about interesting research they've accomplished with coding, or someone has told you that you should learn to code.